src/Neo/SmartContract/Native/NativeContract.cs

```diff
@@ -27,20 +27,20 @@ public abstract class NativeContract
     {
         private class NativeContractsCache
         {
-            public class NativeContractCacheEntry
+            public class CacheEntry
```

Comment:
at least contract cache entry please, cacheentry is not clear

Reply:
Is a subclass, you always call it with the previous one

Reply:
But feel free to change it